src/libs/checkForUpdates.ts

```diff
@@ -0,0 +1,19 @@
+const UPDATE_INTERVAL = 1000 * 60 * 60 * 8;
+
+type CheckForUpdatesParams = {
+    update: () => void;
+    init?: () => void;
+};
+
+function checkForUpdates(platformSpecificUpdater: CheckForUpdatesParams) {
+    if (typeof platformSpecificUpdater.init === 'function') {
+        platformSpecificUpdater.init();
+    }
+
+    // Check for updates every hour
+    setInterval(() => {
+        platformSpecificUpdater.update();
+    }, UPDATE_INTERVAL);
+}
+
+module.exports = checkForUpdates;
```

Comment:
@fabioh8010 @kubabutkiewicz I don't think it was discussed before but don't you think that we should ban module.exports in TS files? Instead use ES6 equivalent:
```suggestion
export default checkForUpdates;
```

We could use one of these [rules](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-commonjs.md) for TS files:
```
'import/no-commonjs': 'error',
'import/no-commonjs': [2, {allowRequire: true}],
```

Reply:
Discussed it with Callstack on slack:

> I told to keep the exports this way because it seems to me that we are using cjs with purpose, the last commit on this file -> https://github.com/Expensify/App/commit/2dbb15c17f89aec15830af1d186faa24ddd7823e
main.js from desktop still uses requires for all imports, so I don’t know if it’s safe to change checkForUpdates.js

We decided to leave such exports for now.